src/main/java/xyz/groundx/caver_ext_kas/kas/kip7/KIP7.java

```diff
@@ -0,0 +1,1459 @@
+     * @return Kip7TokenBalanceResponse
+     * @throws ApiException
+     */
+    public Kip7TokenBalanceResponse allowance(String addressOrAlias, String owner, String spender) throws ApiException {
+        return kip7Api.getTokenAllowance(chainId, addressOrAlias, owner, spender);
+    }
+
+    /**
+     * Shows the amount of tokens approved to the `spender` by the `owner` asynchronously. <br>
+     * GET /v1/contract/{contract-address-or-alias}/account/{owner}/allowance/{spender} <br>
+     *
+     * <pre>Example :
+     * {@code
+     * ApiCallback<Kip7TokenBalanceResponse> callback = new ApiCallback<Kip7TransactionStatusResponse>() {
+     *     ....implements callback method.
+     * }
+     *
+     * String contractAlias = "";
+     * String owner = "";
+     * String spender = "";
+     *
+     * caver.kas.kip7.allowance(contractAlias, owner, spender, callback);
+     * }
+     * </pre>
+     *
+     * @param addressOrAlias Contract address (in hexadecimal with the 0x prefix) or an alias.
+     * @param owner Token owner's Klaytn account address.
+     * @param spender Klaytn account address to search the amount of tokens approved by the owner.
+     * @param callback The callback to handle response.
+     * @return Call
+     * @throws ApiException
+     */
+    public Call allowanceAsync(String addressOrAlias, String owner, String spender, ApiCallback<Kip7TokenBalanceResponse> callback) throws ApiException {
+        return kip7Api.getTokenAllowanceAsync(chainId, addressOrAlias, owner, spender, callback);
+    }
+
+    /**
+     * Shows the `{owner}`'s balance for the given KIP-7 contract. <br>
+     * GET /v1/contract/{contract-address-or-alias}/account/{owner}/balance <br>
+     *
+     * <pre>Example :
+     * {@code
+     * String contractAddress = "";
+     * String owner = "";
+     *
+     * Kip7TokenBalanceResponse response = caver.kas.kip7.balance(testContractAlias, deployerAddress);
+     * }
+     * </pre>
+     *
+     * @param addressOrAlias Contract address (in hexadecimal with the 0x prefix) or an alias.
+     * @param owner Token owner's Klaytn account address.
+     * @return Kip7TokenBalanceResponse
+     * @throws ApiException
+     */
+    public Kip7TokenBalanceResponse balance(String addressOrAlias, String owner) throws ApiException {
+        return kip7Api.getTokenBalance(chainId, addressOrAlias, owner);
+    }
+
+    /**
+     * Shows the `{owner}`'s balance for the given KIP-7 contract asynchronously. <br>
+     * GET /v1/contract/{contract-address-or-alias}/account/{owner}/balance <br>
+     *
+     * <pre>Example :
+     * {@code
+     * ApiCallback<Kip7TokenBalanceResponse> callback = new ApiCallback<Kip7TransactionStatusResponse>() {
+     *     ....implements callback method.
+     * }
+     *
+     * String contractAlias = "";
+     * String owner = "";
+     *
+     * caver.kas.kip7.balanceAsync(contractAlias, owner, callback);
```

Comment:
```suggestion
     * caver.kas.kip7.balanceAsync(contractAlias, owner, callback);
     }
```
There was a javadoc build error. :)